rune/libenclave/internal/runtime/pal/skeleton/README.md

```diff
@@ -64,7 +61,66 @@ Runtimes: rune runc
 ```shell
 docker run -it --rm --runtime=rune \
   -e ENCLAVE_TYPE=intelSgx \
-  -e ENCLAVE_RUNTIME_PATH=/run/rune/liberpal-skeleton.so \
+  -e ENCLAVE_RUNTIME_PATH=/usr/lib/liberpal-skeleton.so \
```

Comment:
Should be /usr/lib64/liberpal-skeleton.so

Reply:
For Ubuntu, it doesn't have `/usr/lib64`, but has '/usr/lib'

Reply:
> Should be /usr/lib64/liberpal-skeleton.so

For Ubuntu, it doesn't have `/usr/lib64`, but has `/usr/lib`

Reply:
> > Should be /usr/lib64/liberpal-skeleton.so
> 
> For Ubuntu, it doesn't have `/usr/lib64`, but has `/usr/lib`

I mean you documented /usr/lib64 in previous section:
```shell
 cd "${path_to_inclavare_containers}/rune/libenclave/internal/runtime/pal/skeleton"
 make
 cp liberpal-skeletion.so /usr/lib64
 ```

So you need to keep consistence, and should document well for both ubuntu and centos.

Reply:
OK

Reply:
> > > Should be /usr/lib64/liberpal-skeleton.so
> > 
> > 
> > For Ubuntu, it doesn't have `/usr/lib64`, but has `/usr/lib`
> 
> I mean you documented /usr/lib64 in previous section:
> 
> ```shell
>  cd "${path_to_inclavare_containers}/rune/libenclave/internal/runtime/pal/skeleton"
>  make
>  cp liberpal-skeletion.so /usr/lib64
> ```
> 
> So you need to keep consistence, and should document well for both ubuntu and centos.

OK